### PR TITLE
Add a non-normative note about how the set of encodings was selected

### PR DESCRIPTION
**Pull request by hsivonen**
Add a note documenting some things mention in #207.

***
<a href="https://whatpr.org/encoding/209.html" title="Last updated on May 6, 2020, 11:41 AM UTC (f204869)">Preview</a> | <a href="https://whatpr.org/encoding/209/037500e...f204869.html" title="Last updated on May 6, 2020, 11:41 AM UTC (f204869)">Diff</a>

### CODE REVIEW

File: encoding.bs [hunk 1]
```diff
@@ -612,6 +612,17 @@ prescribes, as that is necessary to be compatible with deployed content.
...
+<p class=note>The set of supported <a for=/>encodings</a> is primarily based on the intersection
+of the sets supported by major browser engines when the development of this standard started and
+removing encodings that were rarely used legitimately but that could be used in attacks.
+The inclusion of some encodings is questionable in the light of anecdotal evidence of the level
+of use by existing Web content. That is, while they have been broadly supported by browsers,
+it is unclear if they are broadly used by Web content. However, an effort has not been made to
+eagerly remove single-byte encodings that were broadly supported by browsers or are part of the
+ISO 8859 series. In particular, the necessity of the inclusion of <a>IBM866</a>, <a>macintosh</a>,
+<a>x-mac-cyrillic</a>, <a>ISO-8859-3</a>, <a>ISO-8859-10</a>, <a>ISO-8859-14</a>,
+<a>ISO-8859-14</a>, and <a>ISO-8859-16</a> is doubtful for the purpose of supporting existing
```
Comment (ricea) [inline encoding.bs:624]: Is ISO-8859-14 mentioned twice?

Comment (hsivonen) [reply encoding.bs]: Fixed. Thanks.

File: encoding.bs [hunk 2]
```diff
@@ -612,6 +612,17 @@ prescribes, as that is necessary to be compatible with deployed content.
 <a>labels</a> are also available as non-normative
 <a href=encodings.json>encodings.json</a> resource.
 
+<p class=note>The set of supported <a for=/>encodings</a> is primarily based on the intersection
+of the sets supported by major browser engines when the development of this standard started and
+removing encodings that were rarely used legitimately but that could be used in attacks.
```
Comment (ricea) [inline encoding.bs:617]: I found this sentence a little hard to parse. How about

> The set of supported <a for=/>encodings</a> is primarily based on the intersection
> of the sets supported by major browser engines when the development of this standard started, **_while_**
> removing encodings that were rarely used legitimately but that could be used in attacks.

**Comment by annevk** [comment]
Should we remove "active"? At least, I don't think it's really worth removing something at this point.

File: encoding.bs (hunk 2 above)
Comment (hsivonen) [reply encoding.bs:617]: Thanks. Fixed.

**Comment by hsivonen** [comment]
> Should we remove "active"?

Removed. Thanks.

**Comment by annevk** [comment]
@hsivonen I made a couple minor tweaks. If those look alright to you I'll land this.

**Comment by hsivonen** [comment]
LGTM. Thanks.

**Comment by ricea** [comment]
lgtm.
